### PR TITLE
Update pt_BR translation

### PR DESCRIPTION
**Pull request by georgerbr**
Revised and completed.

### CODE REVIEW

File: files/files/translations/locale.pt_BR.properties [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # E agora atualizado para a v.1.68 por Fabio Rodolfo Vertamatti
 # Atualizado para v1.73 por Rodrigo Haas
 # Atualizado para v2.21 por Georger Araújo
-# Revisado e atualizado para as versões de 3.0.0 a v4.0.2 por Gérson Vagner (www.gersonvagner.com) 2017/11/25
+# Revisado e atualizado para v4.4.6 por Georger Araújo
```
Comment (moltenform) [inline files/files/translations/locale.pt_BR.properties:28]: Let's leave a line like `Atualizado para v2.21 por Gérson Vagner (www.gersonvagner.com)`.

**Review by moltenform — CHANGES_REQUESTED** [review]
Thank you! Will merge with the requested change.

File: files/files/translations/locale.pt_BR.properties (hunk 1 above)
Comment (georgerbr) [reply files/files/translations/locale.pt_BR.properties:28]: Yes, that would be fine.

**Comment by moltenform** [comment]
Thank you so much for contributing!

**Pull request merged**